SwiftLint/swiftlint.yml

```diff
@@ -54,6 +55,39 @@ opt_in_rules:
   - identical_operands
   - collection_alignment
   - empty_xctest_method
+  - array_init
+  - contains_over_filter_count
+  - contains_over_filter_is_empty
+  - contains_over_first_not_nil
+  - empty_collection_literal
+  - empty_string
+  - explicit_self
+  - file_name
+  - for_where
+  - joined_default_parameter
+  - last_where
+  - legacy_multiple
+  - literal_expression_end_indentation
+  - lower_acl_than_parent
+  - modifier_order
+  - multiline_arguments
+  - multiline_function_chains
+  - multiline_literal_brackets
+  - multiline_parameters_brackets
```

Comment:
This one is triggered for:
```		
func cell(_ backupObject: BackupObject?,
				  viewController: HDBRRestoreViewController,
				  tableView: UITableView,
				  atIndexPath: IndexPath,
				  restoreProgressBlock: ((_ progress: BackupProgress?) -> Void)? = nil,
				  restoreFinishedBlock: ((_ report: BackupProcessReport?, _ error: BusinessError?) -> Void)? = nil) -> UITableViewCell {
```

To fix it, we would have to write:

```
		func cell(
			_ backupObject: BackupObject?,
				  viewController: HDBRRestoreViewController,
				  tableView: UITableView,
				  atIndexPath: IndexPath,
				  restoreProgressBlock: ((_ progress: BackupProgress?) -> Void)? = nil,
				  restoreFinishedBlock: ((_ report: BackupProcessReport?, _ error: BusinessError?) -> Void)? = nil
			) -> UITableViewCell {
```

Which looks horrible to me